src/public/services/AdminSubmissionsService.ts

```diff
@@ -0,0 +1,84 @@
+import axios from 'axios'
+
+import { EncryptedSubmissionDto, SubmissionMetadataList } from 'src/types'
+import {
+  FormsSubmissionMetadataQueryDto,
+  FormSubmissionMetadataQueryDto,
+  SubmissionCountQueryDto,
+  SubmissionResponseQueryDto,
+} from 'src/types/api'
+
+import { ADMIN_FORM_ENDPOINT } from './AdminFormService'
+
+/**
+ * Counts the number of submissions for a given form
+ * @param urlParameters Mapping of the url parameters to values
+ * @returns The number of form submissions
+ */
+export const countFormSubmissions = async ({
+  formId,
+  dates,
+}: SubmissionCountQueryDto): Promise<number> => {
+  const queryUrl = `${ADMIN_FORM_ENDPOINT}/${formId}/submissions/count`
+  if (dates) {
+    return axios
+      .get(queryUrl, {
+        params: { ...dates },
+      })
+      .then(({ data }) => data)
+  }
+  return axios.get(queryUrl).then(({ data }) => data)
+}
+
+/**
+ * Retrieves the metadata for a page of submissions
+ * @param formId The id of the form to retrieve submission for
+ * @param pageNum The page number of the responses
+ * @returns The metadata of the page of forms
+ */
+export const getFormsMetadata = async ({
```

Comment:
i know the server is named like that (a regret), but should we name this more explicitly like `getFormsMetadataByPage`, very easy to mix up getFormMetadata and getFormsMetadata

Reply:
oml yes pls rename this

Reply:
sure, will rename the other one to `getFormMetadataById` - couldn't think of a good name during that point in time @___@